src/argument_markers.rs

```diff
@@ -60,3 +61,48 @@ impl ArgsKwargs {
         }
     }
 }
+
+static UNDEFINED_CELL: GILOnceCell<UndefinedType> = GILOnceCell::new();
+
+fn get_undefined(_py: Python) -> PyResult<UndefinedType> {
+    Ok(UndefinedType {})
```

Comment:
```suggestion
fn get_undefined() -> UndefinedType {
    UndefinedType {}
```

I don't think you need the result here.